js/piwik.js

```diff
@@ -4960,6 +4965,10 @@ if (typeof window.Matomo !== 'object') {
                 linkTrackingInstalled = false;
                 linkTrackingEnabled   = false;
             };
+            this.disableFingerPrint = function ()
+            {
+                fingerPrintEnable = false;
```

Comment:
I think when `setConsentGiven` or `setCookieConsentGiven` is called, then we can set `fingerPrintEnable = true` as the user has given consent and it should be fine to read those properties